va/drm/va_drm.h

```diff
@@ -40,6 +40,23 @@
 extern "C" {
 #endif
 
+struct driver_name_map {
+    const char *key;
+    int         key_len;
+    const char *name;
+};
+
+static const struct driver_name_map g_driver_name_map[] = {
+    { "i915",       4, "i965"   }, // Intel OTC GenX driver
+    { "pvrsrvkm",   8, "pvr"    }, // Intel UMG PVR driver
+    { "emgd",       4, "emgd"   }, // Intel ECG PVR driver
+    { "hybrid",     6, "hybrid" }, // Intel OTC Hybrid driver
+    { "nouveau",    7, "nouveau"  }, // Mesa Gallium driver
+    { "radeon",     6, "r600"     }, // Mesa Gallium driver
+    { "amdgpu",     6, "radeonsi" }, // Mesa Gallium driver
+    { NULL, }
+};
+
```

Comment:
No - this definitely shouldn't be in a public header.  It probably shouldn't be in a header at all, because the static structure will be duplicated in every file which includes it.